@here/harp-geometry/lib/SubdivisionModifier.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    Face3,
+    Geometry,
+} from "three";
+
+/**
+ * The `SubdivisionModifier` subdivides triangle mesh geometries positioned
+ * on the surface of a sphere centered at `(0, 0, 0)`.
+ */
+export class SubdivisionModifier {
```

Comment:
I would question necessity to put this in separate class with only one method.

Our guidleines say:
> Use simplest tool for the job: function vs. object/namespace/module vs class.

This is plain "data->data" transformation, with no requirements for state (`angle` is hardly _state_).

My proposal `function subdivideGeometryToMatchSphere` ?

Reply:
the API is following the de-facto standard interface of three.js's modifiers.
https://github.com/mrdoob/three.js/tree/dev/examples/js/modifiers